iwa-console/appsettings.json

```diff
@@ -1,28 +1,14 @@
-﻿{
-  "Authentication": {
-    // Azure Cloud instance among:
-    // - AzurePublic (see https://aka.ms/aaddevv2). This is the default value
-    // - AzureUsGovernment (see https://docs.microsoft.com/azure/azure-government/documentation-government-developer-guide)
-    // - AzureChina (see https://docs.microsoft.com/azure/china/china-get-started-developer-guide)
-    // - AzureGermany (See https://docs.microsoft.com/azure/germany/germany-developer-guide)
-    "AzureCloudInstance": "AzurePublic",
-
-    // Azure AD Audience among:
-    // - AzureAdMyOrg (single tenant: you need to also provide the TenantId
-    // - AzureAdMultipleOrgs (multi-tenant): Any work and school accounts
-    // - AzureAdAndPersonalMicrosoftAccount (any work and school account or Microsoft personal account)
-    // - PersonalMicrosoftAccount (Microsoft personal account only)
-    "AadAuthorityAudience": "AzureAdMultipleOrgs",
-
-    // ClientId (ApplicationId) as copied from the application registration (which depends on the cloud instance)
-    // See docs referenced in the AzureCloudInstance section above
-    "ClientId": "ebe2ab4d-12b3-4446-8480-5c3828d04c50"
+{
+  "AzureAd": {
+    "Authority": "https://login.microsoftonline.com/{0}",
+    "TenantId": "[Enter Tenant Id (Obtained from the Azure portal. Select 'Endpoints' from the 'App registrations' blade and use the GUID in any of the URLs), e.g. da41245a5-11b3-996c-00a8-4d99re19f292]",
+    "ClientId": "[Enter the Client Id of the service (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]",
+    "RedirectURI": "ms-appx-web://microsoft.aad.brokerplugin/{0}",

```

Comment:
The redirect uri should be "http://localhost" because this is what MSAL.NET uses on Mac and Linux and old Windows. 
`"ms-appx-web://microsoft.aad.brokerplugin/{0}"` only needs to be set at app registration level.